pep-0563.rst

```diff
@@ -283,8 +283,8 @@ Deprecation policy
 Starting with Python 3.7, a ``__future__`` import is required to use the
 described functionality.  No warnings are raised.
 
-In Python 3.10 this will become the default behavior.  Use of annotations
-incompatible with this PEP is no longer supported.
+This is projected to become the default behavior starting in Python 3.11 or
+later. Use of annotations incompatible with this PEP is no longer supported.
```

Comment:
Since @brettcannon approves I'm okay with this, but we really should wait for the discussion about PEP 649 before we announce any "projections". This may well *never* become the default behavior, if PEP 649 prevails.

Reply:
Would it be appropriate to address the current situation in this PEP or are edits generally limited to corrections?

Reply:
You can mark it with e.g. **Update:** or **Note:**